src/Uno.UWP/Devices/Sensors/SimpleOrientationSensor.cs

```diff
@@ -65,21 +65,6 @@ private SimpleOrientationSensor()
 
 #if __ANDROID__ || __IOS__
 		private void SetCurrentOrientation(SimpleOrientation orientation)
```

Comment:
We may need to validate our own internal use of this class, where assumptions about the Dispatcher may now be wrong. (particularly on android)